### ISSUE TITLE
Git Commits report for Volunteer participation metric

### ISSUE THREAD

**Issue by Archyver** [techlahoma/help-wanted#114]
Make a report showing how many git commits were done over the last month across all Techlahoma repositories and how many different users committing to github for Techlahoma 

Metrics to also consider: How many lines were edited in addition to straight commits

What this doesn't track: How to enable tracking of artistic volunteer Talent. Submitting 1 graphic/design isn't an estimation of the time taken to make that graphic.

This would be used as a metric to show user participation and volunteer efforts for committee meetings

**Comment by carmalou** [comment]
I'm not sure this would help with its intended purpose. Probably 95% or more of our volunteer efforts are not done on github.

**Comment by Archyver** [comment]
Agreed, but I also think it's an easy first step for newer members trying to volunteer, or those like myself, who are not local and wanting to volunteer in some way.

**Comment by kacollins** [comment]
The membership committee has a task to `Establish success metrics to track volunteer involvement`. This seems to be in line with that, but it's probably up to the committee co-chairs @Avasond and @ezrinjaz.
